lessons/evaluating-exponents/langs/en-us/index.adoc

```diff
@@ -0,0 +1,116 @@
+= Evaluating Exponents
+
+@description{Students use Circles of Evaluation analyze the structure of expressions that include exponents.}
+
+@lesson-prereqs{absolute-value-and-opposites-more}
+
+@keywords{exponent, base}
+
+[@lesson-intro-table]
+|===
+
+| Lesson Goals
+| Students will be able to...
+
+* Translate between words, mathematical expressions, and Circles of Evaluation.
+* Evaluate expressions involving exponents and one other operation.
+* Evaluate expressions involving exponents and negations.
+
+
+| Student-facing Lesson Goals
+|
+
+* Let's use Circles of Evaluation to analyze the structure of expressions with exponents.
+
+| Materials
+|[.materials-links]
+@material-links
+
+| Key Points For The Facilitator
+|
+* Circles of Evaluation will not help students to visualize or understand the meaning of exponents as repeated multiplication. Rather, they will help students parse expressions that involve exponents. We recommend ensuring that students are comfortable evaluating exponents before jumping into this lesson.
+
+|===
+
+== Expressions with Exponents and One Operation @duration{30 minutes}
+
+=== Overview
+
+Sudents use Circles, words, and mathematical notation to analyze the structure of expressions with exponents and one operation.
+
+=== Launch
+
+[.lesson-instruction]
+- Show students the following expression: @math{2^3}
+- One way of reading this expression is "two raised to the power of three." Do you know any other ways of reading this expression with an exponent?
+** _Two to the third power; two to the third; two cubed._
+- What does it mean to _cube_ a number?
+** We multiply it by itself three times.
+- What does this expression evaluate to?
+** @math{2^3 = 2 \times 2 \times 2 = 8}
+
+Clarify and reinforce some key concepts about exponents via class discussion:
```

Comment:
I think this misses an opportunity to justify why exponentiation is valuable: _it's a shorthand_. Show the Circle of Eval for 2^3 (or a higher power!), and point out how using exponents makes this much cleaner.

`num-expt(2,8)` is a much nicer way of writing 2 * 2 * 2 * 2 * 2 * 2 * 2 * 2